web/themes/contrib/civictheme/src/Color/CivicthemeColorShadeFilter.php

```diff
@@ -29,7 +29,7 @@ protected function expectedArgumentsCount(): int {
    * @SuppressWarnings(StaticAccess)
    */
   public function filter(string $color): string {
-    return CivicthemeColorUtility::mix($color, '#000', $this->arguments[0]);
+    return CivicthemeColorUtility::mix($color, '#000', (int) $this->arguments[0]);
```

Comment:
@joshua1234511 
We need to asses that the value is numeric before converting it. If it is not - we need to throw an exception.
```
$range = $this->arguments[0];
if (!is_numeric($range)) {
  throw new \Exception(sprintf('Numeric value is expected for range, but %s provided.', $range));
} 
return CivicthemeColorUtility::mix($color, '#000', (int) $range);
```

Reply:
Moved logic to mix, as its the place the check needs to happen. 
Also test is being run on CivicthemeColorUtility::mix and not filter ().